src/lib/transition_frontier_controller_tests/stubs.ml

```diff
@@ -378,6 +378,25 @@ struct
          (root_breadcrumb |> return |> Quickcheck.Generator.return)
          (gen_breadcrumb ~logger ~accounts_with_secret_keys)
 
+  let add_linear_breadcrumbs ~logger ~size ~accounts_with_secret_keys tf =
+    let root = Transition_frontier.root tf in
```

Comment:
Rather than adding breadcrumbs on the root, can it be an arbitrary breadcrumb in the frontier? Can you add an extra argument, state_hash, to add breadcrumbs on a breadcrumb in the transition frontier.

Reply:
instead of passing a hash, I passed a breadcrumb. is this also OK?